tests/system/CLI/CLITest.php

```diff
@@ -73,7 +73,7 @@ public function testWaitZero()
         $time = time();
         CLI::wait(0);
 
-        $this->assertSame(0, time() - $time);
+        $this->assertCloseEnough(0, time() - $time);
```

Comment:
> Failed asserting that 1 is identical to 0.

`time()` returns an interger number.
Does `assertCloseEnough()` help it?

Reply:
What I mean to say is if `time()` is called near the turn of the second, say 1.98 seconds (this will return the whole seconds, e.g. `1687499559`). Then we call `time()` again after executing `CLI::wait(0)` which for example took 5ms, (so `time()` will give something like `1687499560`). So the difference is 1 second. `$this->assertCloseEnough` is just a modified way of asking if the value is less than or equal the tolerance, which in the case of the method, is equal to 1.

Reply:
Okay, `$tolerance = 1`, which means one second in this case.
https://github.com/codeigniter4/CodeIgniter4/blob/ded8d382a054b5e543e3da63e125aa7bb080519d/system/Test/CIUnitTestCase.php#L445